standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/AcidEventListener.java

```diff
@@ -80,22 +82,18 @@ public void onDropTable(DropTableEvent tableEvent) throws MetaException {
       txnHandler.cleanupRecords(HiveObjectType.TABLE, null, table, null, !tableEvent.getDeleteData());
       
       if (!tableEvent.getDeleteData()) {
-        long currentTxn = Optional.ofNullable(tableEvent.getEnvironmentContext())
-          .map(EnvironmentContext::getProperties)
-          .map(prop -> prop.get("txnId"))
-          .map(Long::parseLong)
-          .orElse(0L);
+        long currentTxn = getTxnId(tableEvent.getEnvironmentContext());
         
-        try {
-          if (currentTxn > 0) {
+        if (currentTxn > 0) {
```

Comment:
nit: Why is this change?

Reply:
just a small refactor, moved if out of try block